course7/homeworks/waleed/ls/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++14")
 
 project (LS)
 
-find_package(Boost COMPONENTS filesystem system)
+find_package(Boost COMPONENTS filesystem program_options system)
```

Comment:
I don't know which compiler are you using. but you should set CMAKE_CXX_STANDARD to 14, otherwise other compiles don't know that they should use C++-14 to compile the code.